pages/utilities/image-resizer.tsx

```diff
@@ -0,0 +1,284 @@
+                <Label className="mb-2">Width (px)</Label>
+                <Input
+                  type="number"
+                  placeholder="Enter width"
+                  onChange={handleWidthChange}
+                  value={width ?? ""}
+                  className="mb-2"
+                  disabled={!imageFile}
+                />
+              </div>
+              <div className="flex-1 ml-2">
+                <Label className="mb-2">Height (px)</Label>
+                <Input
+                  typeof="number"
+                  placeholder="Enter height"
+                  onChange={handleHeightChange}
+                  value={height ?? ""}
+                  className="mb-2"
+                  disabled={!imageFile}
+                />
+              </div>
+            </div>
+
+            <div className="flex items-center mb-6 gap-2">
+              <Checkbox
+                id="mantain-aspect-ratio"
+                checked={maintainAspectRatio}
+                onCheckedChange={handleAspectRatioChange}
+                disabled={!imageFile}
+                className="mr-1"
+              />
+              <Label
+                htmlFor="mantain-aspect-ratio"
+                className="mb-0 hover:cursor-pointer"
+              >
+                Maintain Aspect Ratio
+              </Label>
+            </div>
+
+            <Divider />
+
+            <div className="mb-6 flex w-full gap-4">
+              <div className="flex flex-1 flex-col">
+                <Label className="mb-2">Format</Label>
+                <Combobox
+                  data={formatOptions}
+                  onSelect={(value) => setFormat(value as Format)}
+                  defaultValue={format}
+                  disabled={!imageFile}
+                />
+              </div>
+
+              <div className="flex flex-1 flex-col">{qualityInput}</div>
+            </div>
+
+            <Divider />
+
+            <div className={cn(imageFile && "mb-6", "flex w-full gap-4")}>
+              <Button
+                className="flex flex-1"
+                onClick={handleResize}
+                disabled={!imageFile}
+              >
+                Resize
+              </Button>
+
+              <Button
+                disabled={!imageFile}
+                variant="outline"
+                className="flex flex-1"
+              >
+                <DownloadIcon className="h-4 w-4 mr-2" />
+                <a
+                  className={cn(!imageFile && "pointer-events-none")}
+                  href={output}
+                  download={`resized-image.${format}`}
+                >
+                  Download Image
+                </a>
+              </Button>
+            </div>
+
+            {output && (
+              <>
+                <Divider />
+                <div>
+                  <Label>Resized Image</Label>
+                  <div className="flex flex-col flex-1 items-center ring-1 ring-border rounded-lg p-1">
```

Comment:
I think it would be beneficial to users to show resized image size.
This value should be current `width` and `height` of the image that is shown on screen. 
(not existing width and height variables, since they update on change)

```
<div className="flex flex-1 justify-between>
  <Label> Resized Image</Label>
  <Label> 1280 x 640</Label> 
</div>
```